library/components/TNavigationTabs.vue

```diff
@@ -1,16 +1,16 @@
 <template>
   <div
     class="flex flex-wrap items-center bg-white px-5"
-    style="font-family: 'Roboto'"
+    style="font-family: 'Roboto',"
   >
     <a
+      :href="'/' + page.url"
       v-for="page in project.nav"
       :key="page.src"
-      :href="'/' + page.url"
       class="h-12 px-2 pt-3 text-sm border-b-4"
       :class="
         navSelected(
-          page.url,
+          'reporting/'+page.url,
```

Comment:
@sareg0 can I get approval again, had this small fix to make after recent changes to topcoat-core.